config/static/single-pod/pod-main.yaml

```diff
@@ -144,6 +144,12 @@ spec:
       privileged: false
     terminationMessagePath: /dev/termination-log
     terminationMessagePolicy: File
+    # https://kubernetes.io/docs/tasks/configure-pod-container/attach-handler-lifecycle-event/
+    # https://www.freedesktop.org/software/systemd/man/systemd.html#SIGRTMIN+3
+    lifecycle:
+      preStop:
+        exec:
+          command: ["kill", "-s", "SIGRTMIN+3", "1"]
```

Comment:
This is super nice and works like a charm.  A "must have" for a systemd-based container.

One nit (extremely minor): Perhaps `["kill", "-RTMIN+3", "1"]`, to save one line in the YAML output from `oc get -o yaml pod/...`.

Reply:
Updated. Thanks @frasertweedale